crates/re_space_view_spatial/src/ui_3d.rs

```diff
@@ -648,19 +647,22 @@ impl SpatialSpaceView3D {
             );
         }
 
+        // TODO(andreas): Make configurable. Could pick up default radius for this view?
+        let box_radius = Size(re_types::components::Radius::default().0);
```

Comment:
`box_radius` sounds like it's half-extents

```suggestion
        let box_line_radius = Size(re_types::components::Radius::default().0);
```